src/index.ts

```diff
@@ -63,6 +64,8 @@ async function main() {
 
   await routes(server, client, runner);
 
+  stats.increment('.startup');
```

Comment:
Is starting with a `.` normal too?

Reply:
same as above ^ I'm planning to append the `.` to the prefix if it exists when the StatsD client is initialised, rather than have `.`s added everywhere.